OpTestConfiguration.py

```diff
@@ -107,6 +107,10 @@ def get_parser():
     hostinstgroup.add_argument("--proxy", default="", help="proxy for the Host to access the internet. "
                                "Only needed for tests that install an OS")
 
+    hostcmdgroup = parser.add_argument_group('Host Run Commands', 'Options for Running custom commands on the Host')
+    hostcmdgroup.add_argument("--host-cmd", help="Command to run", default="")
+    hostcmdgroup.add_argument("--host-cmd-file", help="Commands to run from file", default="")
+    hostcmdgroup.add_argument("--host-cmd-timeout", help="Timeout for command", default=1000)
```

Comment:
Can you add type as int, otherwise timeout is taking as string and failing.
```
======================================================================
ERROR [4.312s]: runTest (testcases.RunHostTest.RunHostTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins_data/sath/pridhivi/upstream_optest/sathnaga/op-test-framework/testcases/RunHostTest.py", line 45, in runTest
    con.run_command(self.host_cmd, timeout=self.host_cmd_timeout)
  File "/home/jenkins_data/sath/pridhivi/upstream_optest/sathnaga/op-test-framework/common/OpTestIPMI.py", line 264, in run_command
    rc = console.expect([BMC_DISCONNECT, "\[console-pexpect\]#$"], timeout=timeout)
  File "/home/jenkins_data/sath/pridhivi/upstream_optest/sathnaga/op-test-framework/common/OPexpect.py", line 84, in expect
    searchwindowsize=searchwindowsize)
  File "/usr/lib/python2.7/site-packages/pexpect/__init__.py", line 1418, in expect
    timeout, searchwindowsize)
  File "/usr/lib/python2.7/site-packages/pexpect/__init__.py", line 1433, in expect_list
    timeout, searchwindowsize)
  File "/usr/lib/python2.7/site-packages/pexpect/__init__.py", line 1481, in expect_loop
    end_time = time.time() + timeout
TypeError: unsupported operand type(s) for +: 'float' and 'str'
```

Reply:
done...